README.md

```diff
@@ -1,6 +1,10 @@
 <!-- vscode-markdown-toc -->
 # O-RAN O2IMS
 
+<!-- markdownlint-disable MD033 -->
+<img src="https://img.shields.io/badge/O--RAN_Compliance-59%25-yellow"/>
```

Comment:
The image says "O-RAN Compliance" - what does that mean? There are many different O-RAN interface specifications and we won't be compliant to all of them (or at least not all of them apply to our implementation). See https://www.o-ran.org/certification-badging for info.

Reply:
Updated to identify the specific tests being referenced and linked to the test repo. Thank you!